package.json

```diff
@@ -9,6 +9,8 @@
     "@mui/icons-material": "^5.11.11",
     "@mui/lab": "^5.0.0-alpha.58",
     "@mui/material": "^5.11.13",
+    "@mui/x-tree-view": "^6.17.0",
```

Comment:
I can't comment on the actual line, 2 lines up, but at a glance I don't see any references in the code to `@mui/lab` anymore - unless I'm missing something, should we remove that?

Reply:
This is a good catch -- updated